src/pykeen/nn/node_piece.py

```diff
@@ -6,6 +6,7 @@
 from typing import Callable, Iterable, Optional, Sequence, Tuple, Union
 
 import numpy
+import numpy as np
```

Comment:
`numpy` was already imported (with its proper, and short-enough, name 😉 )

Reply:
yes yes, I have already removed that in https://github.com/pykeen/pykeen/pull/776/commits/65c7a85dcaa20701721ecc1a24766ca5ff0d81f1